core-primitives/stf-executor/src/executor.rs

```diff
@@ -186,10 +189,24 @@ where
 			let (state_lock, mut state) = self.state_handler.load_for_mutation(&shard_id)?;
 			match Stf::update_parentchain_block(&mut state, header.clone()) {
 				Ok(_) => {
-					self.state_handler.write_after_mutation(state, state_lock, &shard_id)?;
+					//self.state_handler.write_after_mutation(state, state_lock, &shard_id)?;
```

Comment:
Is this intentional?

Reply:
Yes. Actually I should remove it ;)
The state is now saved after the update of the ceremony phase storage